tests/trx_generator/main.cpp

```diff
@@ -45,33 +45,26 @@ struct signed_transaction_w_signer {
    fc::crypto::private_key _signer;
```

Comment:
`trx` could be moved into `_trx`.

Reply:
Addressed: 42c5900bc2bc06b14da3f3879778b476b6f86c3f